lib/event-libs/sdevent/CMakeLists.txt

```
@@ -0,0 +1,37 @@
+# The strategy is to only export to PARENT_SCOPE
+#
+#  - changes to LIB_LIST
+#  - includes via include_directories
+#
+# and keep everything else private
+
+include_directories(.)
+
+# configure or find systemd library
+set(LIB_SYSTEMD_LIBRARIES CACHE PATH "Path to the libsystemd library")
+if ("${LWS_SYSTEMD_LIBRARIES}" STREQUAL "")
+    if (NOT LIB_SYSTEMD_FOUND)
+        message("searching for systemd")
+        find_library(LIBSYSTEMD_LIBRARIES NAMES systemd)
+    endif()
+else()
+    set(LIBSYSTEMD_LIBRARIES ${LWS_SYSTEMD_LIBRARIES})
+endif()
+message("libsystemd libraries: ${LIBSYSTEMD_LIBRARIES}")
+
+if (LWS_WITH_EVLIB_PLUGINS)
+
+    create_evlib_plugin(evlib_sd sdevent.c ${LIBSYSTEMD_LIBRARIES})
```

Other implementations do also mention their private header here.

```suggestion
    create_evlib_plugin(evlib_sd sdevent.c private-lib-event-libs-sdevent.h ${LIBSYSTEMD_LIBRARIES})
```

---

Fixed.